src/main/kotlin/ar/edu/unahur/obj2/vendedores/CentroDeDistribucion.kt

```diff
@@ -0,0 +1,26 @@
+package ar.edu.unahur.obj2.vendedores
+
+import java.lang.Exception
+
+class CentroDeDistribucion(val ciudad: Ciudad) {
+
+    val vendedores = mutableListOf<Vendedor>()
+
+    fun agregarVendedor(vendedor: Vendedor) {
+        if (!vendedores.contains(vendedor)) {
+            vendedores.add(vendedor)
+        }
+        else {
+            throw Exception("El vendedor ya se encuentra registrado")
+        }
+    }
```

Comment:
No está mal, pero una buena practica es comparar la condición que rompe y lanzar la excepción
```suggestion
    fun agregarVendedor(vendedor: Vendedor) {
        if (vendedores.contains(vendedor)) {
            throw Exception("El vendedor ya se encuentra registrado")
        }
        vendedores.add(vendedor)
    }
```
De esta forma si el vendedor ya está agregado, va a romper antes de agregarlo.